### ISSUE TITLE
Information related to existing defenses

### ISSUE THREAD

**Issue by SravantiAddepalli** [fra31/auto-attack#5]
Dear authors,

Thank you for releasing the exhaustive list of state-of-the-art defenses along with their evaluations on a standard set of benchmark attacks. This is very helpful to the research community working in this area. 
Could you please share more information related to the models listed on the leaderboard? Specifically, it would be helpful if you can share information related to the fields that we need to fill up while submitting a new defense such as architecture, link to code, link to model weights and a brief description of the defense. 

Thanks,
Sravanti

**Comment by fra31** [comment]
Hi Sravanti,

thanks for the interest in our work!
More details about (most of) the models currently in the list can be found in the paper.

About submitting new defenses, the [template](https://github.com/fra31/auto-attack/issues/new?assignees=&labels=&template=add-defense.md&title=Add+%5Bdefense+name%5D) is meant to provide some more information about the models tested. For example, factors like size of the network, training data, etc. can influence the final robustness, and knowing these details might be important to compare different techniques. Also some papers have goals other than state-of-the-art robustness, e.g. faster training.
The template is just a guideline: it's important to provide the weights of a pretrained model (checkpoints) and code to restore and use it (you can provide e.g. a link to a GitHub page, a shared folder...), so that we can run the attack. You can also provide the robust accuracy you get with your evaluation (just please specify the attack used -- even if you've already run AutoAttack), the architecture (e.g. ResNet-18) and a description (whatever you consider important) of the technique proposed. It's not necessary to fill all the fields.

Let me know if you have other questions!

**Comment by SravantiAddepalli** [comment]
Great, thank you for the information.

Regards,
Sravanti
